<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatplotlibDemo</class>
 <widget class="QMainWindow" name="MatplotlibDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matplotlib Demo with Titanic Data</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="mpl_widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>801</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_plot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_quit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_clear">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MatplotlibDemo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_quit</sender>
   <signal>clicked()</signal>
   <receiver>MatplotlibDemo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
